Log4swift/LoggerFactory+loadFromFile.swift

```diff
@@ -148,14 +148,18 @@ extension LoggerFactory : FileObserverDelegate {
   }
   
   private func appenderForClassName(_ className: String) -> Appender.Type? {
+    #if !os(watchOS)
```

Comment:
Hi Igor,

I would rather have this written like that :
```
let nsLoggerAppenderType: Appender.Type?
#if os(watchOS)
  nsLoggerAppenderType = nil
#else
  nsLoggerAppenderType = NSLoggerAppender.self
#endif
```

and then in the switch case bellow, use this new constant.
This would allow for a consistent switch case construct grouping all possible cases, with preconditions grouped before.
I  believe it will be more readable.

What do you think ?